sources/packages/web/src/views/aest/institution/ApplicationOfferingChangeRequestForm.vue

```diff
@@ -0,0 +1,165 @@
+<template>
+  <full-page-container layout-template="centered-card-tab">
+    <template #header>
+      <header-navigator
+        title="Assessments"
+        :routeLocation="goBackRouteParams"
+        subTitle="View Request"
+      >
+        <template
+          #buttons
+          v-if="
+            applicationOfferingChangeRequestDetails.status ===
+            ApplicationOfferingChangeRequestStatus.InProgressWithSABC
+          "
+        >
+          <v-row class="p-0 m-0">
+            <v-btn color="primary" variant="outlined"
+              >Decline reassessment</v-btn
+            >
+            <v-btn class="ml-2" color="primary">Approve reassessment</v-btn>
+          </v-row>
+        </template>
+      </header-navigator>
+      <application-offering-change-details-header
+        class="m-4"
+        :headerDetails="headerDetails"
+      />
+    </template>
+    <template
+      #alerts
+      v-if="
+        applicationOfferingChangeRequestDetails.status ===
+        ApplicationOfferingChangeRequestStatus.InProgressWithStudent
+      "
+      ><application-offering-change-request-banner />
+    </template>
+    <template #tab-header>
+      <student-application-offering-change-details
+        class="mb-6"
+        :studentName="applicationOfferingChangeRequestDetails.studentFullName"
+        :applicationNumber="
+          applicationOfferingChangeRequestDetails.applicationNumber
+        "
+        :locationName="applicationOfferingChangeRequestDetails.locationName"
+        :reasonForChange="applicationOfferingChangeRequestDetails.reason"
+        :accessedNoteDescription="
+          applicationOfferingChangeRequestDetails.assessedNoteDescription
+        "
+        :applicationOfferingChangeRequestStatus="
+          applicationOfferingChangeRequestDetails.status
+        "
```

Comment:
I see, all the variables are retrieved from `applicationOfferingChangeRequestDetails` and passed, and this component is only used for this `.vue` file, so instead of passing it separately how about passing the object itself?

Reply:
Felt the same IMO number props shouldn't be so much.

Reply:
6 out of the 14 properties are passed to the `student-application-offering-change-details` component. I remember having a similar conversation of passing only the required props to a component with @andrewsignori-aot earlier. Please correct me if I am missing something.

Reply:
I would need more information about it @sh16011993.
What @ann-aot mentioned about this being a dedicated component and the number of properties makes sense.
The property in the component does not need to be the entire object, it can be just a `Pick` of the properties required by the component.
I am not sure if the question is about sending an object with, for instance, 20 properties, to use just a few. If this is the case we should avoid it but as mentioned earlier, having the prop defined with the specific properties would satisfy what @ann-aot suggestion and @dheepak-aot also agreed.

Reply:
I have made a change to the code @andrewsignori-aot . If it is not what is expected, we can discuss.